### ISSUE TITLE
Margin-top not set to null

### ISSUE THREAD

**Issue by at-benni** [stellarwp/kadence-blocks#451]
Hi,
when inserting a page template from your template library, between the sections (rowlayout) there is a blank space, this is because unlike the marin-bottom the margin-top is not set to 0.
This is a completely fresh WordPress Installation with only the kadence plugin activated.
![grafik](https://github.com/stellarwp/kadence-blocks/assets/109149472/47d10697-b8e2-4022-9d55-25b7c015d934)

**Comment by kadencewp** [comment]
Hey, Looks like you are using a theme that is applying block spacing (a space added between top level blocks) you can control this in the site styles of your theme.

**Comment by at-benni** [comment]
No, I am not using a Theme. As I said "This is a completely fresh WordPress Installation with only the kadence plugin activated."
@kadencewp
